frontend/packages/process-editor/src/components/ConfigPanel/ConfigContent/ConfigContent.tsx

```diff
@@ -58,14 +61,20 @@ export const ConfigContent = (): React.ReactElement => {
           className={classes.displayTile}
           showPadlock={false}
         />
-        {isSigningTask && (
+        {(isSigningTask || isUserControlledSigningTask) && (
```

Comment:
What do you think about extracting `(isSigningTask || isUserControlledSigningTask)` into a variable to improve readability?